lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -333,6 +343,17 @@ def translate_hash_to_trigger(puppet_trigger)
       self.fail "Unknown schedule type: #{puppet_trigger["schedule"].inspect}"
     end
 
+    if puppet_trigger['minutes_interval']
+      integer_interval = Integer(puppet_trigger['minutes_interval'])
+      self.fail 'minutes_interval must be an integer greater or equal to 0' unless integer_interval >= 0
```

Comment:
I'm not sure what you mean, I would like to leave it as it is and you can modify the whole file completely into the new coding style someday. The unless start_date >= Date.new(1753, 1, 1) was just an example of what's already in there, I don't want to change anything on that old code.